source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -32,315 +32,136 @@ The procedure on this page documents the following:
+  This can be any single node, or a load balancer/proxy which handles connections to the MinIO nodes.
 
-         minio_node_drive_io_waiting{job-"minio-job"}[5m]
+  .. cond:: k8s
 
-   .. tab-item:: Recommended Metrics
+     For Prometheus deployments in the same cluster as the MinIO Tenant, you can specify the service DNS name for the ``minio`` service.
 
-      MinIO recommends the following as a basic set of metrics to monitor.
+     For Prometheus deployments external to the cluster, you must specify an ingress or load balancer endpoint configured to route connections to and from the MinIO Tenant.
 
-      See :ref:`minio-metrics-and-alerts` for information about all available metrics.
+Restart Prometheus with the updated configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      .. list-table::
-         :header-rows: 1
-         :widths: 40 60
-         :width: 100%
+Add the desired ``scrape_configs`` jobs to your Prometheus configuration file and start the Prometheus cluster:
 
-         * - Metric
-           - Description
-     
-         * - ``minio_node_drive_free_bytes``
-           - Total storage available on a drive.
+.. code-block:: shell
+   :class: copyable
 
-         * - ``minio_node_drive_free_inodes``
-           - Total free inodes.
+   prometheus --config.file=prometheus.yaml
 
-         * - ``minio_node_drive_latency_us``
-           - Average last minute latency in µs for drive API storage operations.
 
-         * - ``minio_node_drive_offline_total``
-           - Total drives offline in this node.
+Analyze collected metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-         * - ``minio_node_drive_online_total``
-           - Total drives online in this node.
+Prometheus includes an :prometheus-docs:`expression browser <prometheus/latest/getting_started/#using-the-expression-browser>`. 
+You can execute queries here to analyze the collected metrics.
 
-         * - ``minio_node_drive_total``
-           - Total drives in this node.
+The following query examples return metrics collected by Prometheus every five minutes for a scrape job named ``minio-job``:
 
-         * - ``minio_node_drive_total_bytes``
-           - Total storage on a drive.
+.. code-block:: shell
+   :class: copyable
 
-         * - ``minio_node_drive_used_bytes``
-           - Total storage used on a drive.
+   minio_system_drive_used_bytes{job-"minio-job"}[5m]
+   minio_system_drive_used_inodes{job-"minio-job"}[5m]
 
-         * - ``minio_node_drive_errors_timeout``
-           - Total number of drive timeout errors since server start.
+   minio_cluster_usage_buckets_total_bytes{job-"minio-job"}[5m]
+   minio_cluster_usage_buckets_objects_count{job-"minio-job"}[5m]
 
-         * - ``minio_node_drive_errors_availability``
-           - Total number of drive I/O errors, permission denied and timeouts since server start.
+   minio_api_requests_total{job-"minio-job"}[5m]
+   minio_api_requests_errors_total{job-"minio-job"}[5m]
```

Comment:
Replaced the original v2 metrics with something that looked likely from v3. Are there better examples?